swag/db.py

```diff
@@ -85,24 +108,64 @@ def add_user(self, user, guild):
         else:
             raise AccountAlreadyExist
 
+    def add_cagnotte(self, cagnotte_name, cagnotte_currency, cagnotte_creator_id):
+        if cagnotte_name not in self.cagnotte_id:
+            self.cagnotte_id[cagnotte_name] = self.next_cagnotte_id
+            self.next_cagnotte_id += 1
+
+            self.cagnotte_name.append(cagnotte_name)
+            if cagnotte_currency == Currency.SWAG:
+                self.cagnotte_balance.append(0)
+            elif cagnotte_currency == Currency.STYLE:
+                self.cagnotte_balance.append(Decimal(0))
+
+            self.cagnotte_currency.append(cagnotte_currency)
+            self.cagnotte_manager.append([cagnotte_creator_id])
+            self.cagnotte_participant.append(set())
```

Comment:
`{}` instead of `set()`

Reply:
Also `_participants`

Reply:
why {} ? It can be confused with dictionnary

Reply:
`set()` is better, nevermind,`{}` is a dict